airflow/providers/amazon/aws/operators/sagemaker.py

```diff
@@ -42,9 +43,19 @@
 
 
 def serialize(result: dict) -> str:
+    logging.getLogger(__name__).error(result)
     return json.loads(json.dumps(result, cls=AirflowJsonEncoder))
 
 
+def path_to_s3_dataset(path):
+    from openlineage.client.run import Dataset
```

Comment:
I'm not seeing a reason for this to be imported locally down here?

Reply:
We don't want to require people to have OpenLineage installed to use Sagemaker, or AWS provider - importing this locally helps us avoid that.